h/templates/activity/search.html.jinja2

```diff
@@ -73,6 +68,17 @@
       </li>
     {% endfor %}
   </ul>
+  {% if bucket.uri %}
+    <div class="search-bucket-stats__key">
+      {{ svg_icon('up-right-arrow', css_class='search-bucket-stats__icon') }}
+      URL
```

Comment:
It probably ought to be. I'm guessing @lenazun would have a better idea if "URL" would ever be changed by translation.